packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -997,7 +1036,8 @@ class DescopeWc extends BaseDescopeWc {
           ...eleDescopeAttrs,
           ...formData,
           // 'origin' is required to start webauthn. For now we'll add it to every request
-          origin: window.location.origin,
+          origin:
+            this.flowState.current.originOverride || window.location.origin,
```

Comment:
Where do origin override comes from ?

Reply:
Overridden by the Android SDK in case the native code does the WebAuthn instead of the webpage, in which case the origin value isn't the URL, but rather a signature of the running app